src/main/java/org/jabref/gui/backup/BackupChoiceDialog.java

```diff
@@ -0,0 +1,73 @@
+package org.jabref.gui.backup;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ButtonBar;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Label;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.layout.VBox;
+
+import org.jabref.gui.util.BaseDialog;
+import org.jabref.logic.l10n.Localization;
+
+public class BackupChoiceDialog extends BaseDialog<BackupChoiceDialogRecord> {
+    public static final ButtonType RESTORE_BACKUP = new ButtonType(Localization.lang("Restore from backup"), ButtonBar.ButtonData.OK_DONE);
+    public static final ButtonType IGNORE_BACKUP = new ButtonType(Localization.lang("Ignore backup"), ButtonBar.ButtonData.CANCEL_CLOSE);
+    public static final ButtonType REVIEW_BACKUP = new ButtonType(Localization.lang("Review backup"), ButtonBar.ButtonData.LEFT);
+
+    private final ObservableList<BackupEntry> tableData = FXCollections.observableArrayList();
+
+    private final Path backupDir;
+    @FXML
+    private final TableView<BackupEntry> backupTableView;
+
+    public BackupChoiceDialog(Path backupDir, List<BackupEntry> backups) {
+        this.backupDir = backupDir;
+
+        setTitle(Localization.lang("Choose backup file"));
+        setHeaderText(null);
+        getDialogPane().setMinHeight(150);
+        getDialogPane().setMinWidth(450);
+        getDialogPane().getButtonTypes().setAll(RESTORE_BACKUP, IGNORE_BACKUP, REVIEW_BACKUP);
+
+        String content = Localization.lang("It looks like JabRef did not shut down cleanly last time the file was used.") + "\n\n" +
+                Localization.lang("Do you want to recover the library from a backup file?");
```

Comment:
Merge this into one string!

(Even though in the original, newlines were also split like this, but this is wrong)